Engine/Plugins/Marketplace/RuntimeMeshImportExport/Source/RuntimeMeshImportExport/Private/RuntimeMeshImportExport.cpp

```diff
@@ -16,11 +16,11 @@ void FRuntimeMeshImportExportModule::StartupModule()
 	// This code will execute after your module is loaded into memory; the exact timing is specified in the .uplugin file per-module
 	FString PluginBaseDir = IPluginManager::Get().FindPlugin("RuntimeMeshImportExport")->GetBaseDir();
 	FString configString;
-#if UE_BUILD_SHIPPING
+
+	//AMCHANGE_begin
+	//#AMCHANGE Always use the release DLLs, not all testers and QA always have all the dev dependencies
```

Comment:
`dev dependencies` is not a very clear way to describe it.

I would propose: `not all testers and QA always have the debug dll's required to load the debug assimp dll.`